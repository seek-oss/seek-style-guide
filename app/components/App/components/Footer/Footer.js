import styles from './Footer.less';

import React from 'react';
import GridContainer from 'GridContainer/GridContainer';
import GithubIcon from './GithubIcon/GithubIcon';

export default function Footer() {
  return (
    <GridContainer>
      <div className={styles.root}>
        <a
          className={styles.link}
          href="https://github.com/SEEK-Jobs/seek-style-guide"
          title="View project on Github">
          <GithubIcon svgClassName={styles.github} />
        </a>
      </div>
    </GridContainer>
  );
}
