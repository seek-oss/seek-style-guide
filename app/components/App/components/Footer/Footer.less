@import (reference) "~seek-style-guide/theme";

.root {
  .touchableText();
  text-align: center;
  margin: (@grid-row-height * 4);
}

.github {
  fill: @sk-mid-grey-dark;
  opacity: 0.5;
  transition: opacity .3s ease;
  .link:hover & {
    opacity: 1;
  }
}
