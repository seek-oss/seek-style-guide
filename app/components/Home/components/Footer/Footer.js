import styles from './Footer.less';
import github from './github.svg';

import React from 'react';
import GridContainer from 'GridContainer/GridContainer';

export default function Footer() {
  const markup = github.replace('<svg ', `<svg class="${styles.github}" `);

  return (
    <GridContainer>
      <div className={styles.root}>
        <a
          className={styles.link}
          href="https://github.com/SEEK-Jobs/seek-style-guide"
          title="View project on Github"
          dangerouslySetInnerHTML={{ __html: markup }} // eslint-disable-line react/no-danger
        />
      </div>
    </GridContainer>
  );
}
