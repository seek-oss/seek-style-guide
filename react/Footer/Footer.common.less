@import (reference) "~seek-style-guide/theme";

@footer-vertical: ~"and (max-width: 767px)";
@footer-horizontal: ~"and (min-width: 768px)";
@footer-classic: ~"and (min-width: 1025px)";

.footerText() {
  .rawText();
  line-height: (@grid-row-height * 4);
  height: (@grid-row-height * 4);
}

.footerLink() {
  .link();
  color: @sk-link-on-mid-gray-light;

  @media only screen @footer-vertical {
    .touchableText();
  }

  @media only screen @footer-horizontal {
    .footerText();
    display: inline-block;
  }
}
