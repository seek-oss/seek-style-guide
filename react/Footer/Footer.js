import styles from './Footer.less';

import tools, { seekSites, seekApps } from './data/tools';
import company, { partners, services } from './data/company';
import connect, { social } from './data/connect';
import employers from './data/employers';
import copyright from './data/copyright';

import classnames from 'classnames';
import React, { Component } from 'react';
import PropTypes from 'prop-types';

import FooterLink from './FooterLink/FooterLink';
import FooterNav from './FooterNav/FooterNav';
import Hidden from '../Hidden/Hidden';
import ToggleContainer from './ToggleContainer/ToggleContainer';
import {
  AUTHENTICATED,
  UNAUTHENTICATED,
  AUTH_PENDING
} from '../private/authStatusTypes';

const defaultLinkRenderer = props => <a {...props} />;

export default class Footer extends Component {
  static propTypes = {
    locale: PropTypes.oneOf(['AU', 'NZ']),
    linkRenderer: PropTypes.func,
    authenticationStatus: PropTypes.oneOf([
      AUTHENTICATED,
      UNAUTHENTICATED,
      AUTH_PENDING
    ])
  };

  static defaultProps = {
    locale: 'AU',
    linkRenderer: defaultLinkRenderer,
    authenticationStatus: AUTH_PENDING
  };

  constructor() {
    super();

    this.renderLink = this.renderLink.bind(this);
  }

  shouldComponentUpdate(nextProps) {
    return (
      this.props.locale !== nextProps.locale ||
      this.props.authenticationStatus !== nextProps.authenticationStatus
    );
  }

  renderLink({ name, specificLocale, secondary, ...restProps }, i) {
    const { locale, linkRenderer, authenticationStatus } = this.props;

    return !specificLocale || specificLocale === locale ? (
      <FooterLink
        secondary={secondary}
        linkRenderer={linkRenderer}
        authenticationStatus={authenticationStatus}
        key={i}
        {...restProps}
      >
        {name}
      </FooterLink>
    ) : null;
  }

  render() {
    const { linkRenderer } = this.props;

    return (
      <footer role="contentinfo" className={styles.root}>
        <Hidden print>
          <div className={styles.content}>
            <div className={styles.columns}>
              <FooterNav label="Tools">
                {tools.map(this.renderLink)}
                <ToggleContainer
                  name="DownloadAppsToggle"
                  label="Download apps"
                >
                  {seekApps.map(this.renderLink)}
                </ToggleContainer>
                <ToggleContainer name="PartnerSitesToggle" label="SEEK sites">
                  {seekSites.map(this.renderLink)}
                </ToggleContainer>
              </FooterNav>

              <FooterNav secondary label="Company">
                {company.map(this.renderLink)}
                <ToggleContainer
                  secondary
                  name="InternationalPartnersToggle"
                  label="International partners"
                >
                  {partners.map(this.renderLink)}
                </ToggleContainer>
                <ToggleContainer
                  secondary
                  name="PartnerServicesToggle"
                  label="Partner services"
                  data-automation="partner-services-toggle"
                >
                  {services.map(this.renderLink)}
                </ToggleContainer>
              </FooterNav>

              <FooterNav label="Connect">
                {connect.map(this.renderLink)}
                <ToggleContainer name="SocialToggle" label="Social">
                  {social.map(this.renderLink)}
                </ToggleContainer>
              </FooterNav>

              <FooterNav secondary label="Employers">
                {employers.map(this.renderLink)}
              </FooterNav>
            </div>

            <div className={styles.copyright}>
              {copyright.map(({ name, analytics, href, rel, secondary }, key) =>
                linkRenderer({
                  children: name,
                  href,
                  rel,
                  'data-analytics': analytics,
                  key,
                  className: classnames({
                    [styles.copyrightLink]: true,
                    [styles.secondaryLink]: secondary
                  })
                })
              )}
              <p className={styles.copyrightMessage}>
                {'\u00A9 SEEK. All rights reserved.'}
              </p>
            </div>
          </div>
        </Hidden>
      </footer>
    );
  }
}
