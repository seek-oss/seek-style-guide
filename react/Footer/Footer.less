@import (reference) "~seek-style-guide/theme";
@import (reference) "./Footer.common.less";

.root {
  .classicFontStack();
  background-color: @sk-footer;
  width: 100%;
}

.content {
  @media only screen @footer-vertical {
    padding: (@grid-row-height * 2) 0;
  }

  @media only screen @footer-horizontal {
    padding: (@grid-row-height * 2) 0 (@grid-row-height * 3);
  }
}

.columns {
  @media only screen @footer-horizontal {
    display: flex;
  }
}

.copyright {
  @media only screen @footer-vertical {
    display: flex;
    flex-direction: column;
    flex: 1 0 100%;
    justify-content: flex-start;
  }

  @media only screen @footer-horizontal {
    margin-top: (@grid-row-height * 3);
  }
}

.copyrightLink {
  .footerLink();

  @media only screen @footer-horizontal {
    margin-right: @grid-gutter-width;
  }
}

.copyright .secondaryLink {
  @media only screen @footer-vertical {
    display: none;
  }
}

.copyrightMessage {
  @media only screen @footer-horizontal {
    .footerText();
    display: inline-block;
  }
  @media only screen @footer-vertical {
    .touchableText(@standard-type-scale);
  }
}
