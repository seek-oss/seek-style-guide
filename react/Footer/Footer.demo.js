import { Footer } from 'seek-style-guide/react';
import * as sketch from './Footer.sketch';
import { makeDummyLinkRendererForPath } from '../Header/Header.demo';

const ROUTE = '/footer';

export default {
  route: '/footer',
  title: 'Footer',
  category: 'Layout',
  component: Footer,
  sketch,
  initialProps: {
    authenticationStatus: 'authenticated',
    linkRenderer: makeDummyLinkRendererForPath(ROUTE)
  },
  options: [
    {
      label: 'Locale',
      type: 'radio',
      states: [
        {
          label: 'AU',
          transformProps: props => props
        },
        {
          label: 'NZ',
          transformProps: props => ({
            ...props,
            locale: 'NZ'
          })
        }
      ]
    },
    {
      label: 'Authentication',
      type: 'radio',
      states: [
        {
          label: 'Authenticated',
          transformProps: props => props
        },
        {
          label: 'Unauthenticated',
          transformProps: props => ({
            ...props,
            authenticationStatus: 'unauthenticated'
          })
        },
        {
          label: 'Pending',
          transformProps: props => ({
            ...props,
            authenticationStatus: 'pending'
          })
        }
      ]
    }
  ]
};
