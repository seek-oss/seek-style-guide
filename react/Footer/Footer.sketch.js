import React from 'react';
import PropTypes from 'prop-types';
import Footer from './Footer';

const DummyLinkRenderer = ({ href, ...props }) => <a {...props} />;

DummyLinkRenderer.propTypes = {
  href: PropTypes.string
};

export const blockSymbols = {
  'Footer/Australia': <Footer linkRenderer={DummyLinkRenderer} />,
  'Footer/New Zealand': <Footer locale="NZ" linkRenderer={DummyLinkRenderer} />
};
