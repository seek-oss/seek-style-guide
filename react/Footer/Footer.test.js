import React from 'react';
import { render } from 'enzyme';

import Footer from './Footer';

const renderFooter = props => render(<Footer {...props} />);

describe('Footer:', () => {
  it('should render with locale of AU', () => {
    expect(renderFooter({ locale: 'AU' })).toMatchSnapshot();
  });

  it('should render with locale of NZ', () => {
    expect(renderFooter({ locale: 'NZ' })).toMatchSnapshot();
  });

  it('should render when authenticated', () => {
    expect(
      renderFooter({ authenticationStatus: 'authenticated' })
    ).toMatchSnapshot();
  });

  it('should render when unauthenticated', () => {
    expect(
      renderFooter({ authenticationStatus: 'unauthenticated' })
    ).toMatchSnapshot();
  });

  it('should render when authentication is pending', () => {
    expect(renderFooter({ authenticationStatus: 'pending' })).toMatchSnapshot();
  });
});
