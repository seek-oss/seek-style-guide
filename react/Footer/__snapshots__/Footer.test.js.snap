// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer: should render with locale of AU 1`] = `
<footer
  aria-labelledby="FooterHeading"
  className="root"
  role="contentinfo"
>
  <span
    className="print"
  >
    <section>
      <span
        className="root"
      >
        <h1
          id="FooterHeading"
        >
          Footer
        </h1>
      </span>
      <div
        className="content"
      >
        <nav
          aria-label="Tools"
          className="category"
        >
          <h1
            className="heading"
          >
            Tools
          </h1>
          <ul>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:profile"
                href="/profile/"
              >
                Profile
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:my+activity"
                href="/myactivity#favourite"
              >
                Saved searches
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:favourite"
                href="/my-activity/saved-jobs"
              >
                Saved jobs
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:applied"
                href="/my-activity/applied-jobs"
              >
                Applied jobs
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:advice+tips"
                href="/career-advice/"
              >
                Advice & tips
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:companies"
                href="/companies/"
              >
                Company reviews
              </a>
            </li>
          </ul>
        </nav>
        <nav
          aria-label="Company"
          className="category"
        >
          <h1
            className="heading"
          >
            Company
          </h1>
          <ul>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:about+seek"
                href="/investor/about-us"
              >
                About SEEK
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:media"
                href="/investor/news-media"
              >
                Media
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:work+for+seek"
                href="/work-for-seek"
              >
                Work for SEEK
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:investor+centre"
                href="https://ir.seek.com.au"
              >
                Investor centre
              </a>
            </li>
            <li>
              <input
                className="toggle"
                data-automation={undefined}
                id="InternationalPartnersToggle"
                type="checkbox"
              />
              <label
                className="toggleLink toggleLabel"
                htmlFor="InternationalPartnersToggle"
              >
                International partners
                <span
                  className="root root down chevron"
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "mock svg",
                    }
                  }
                />
              </label>
              <ul
                className="toggleContainer"
              >
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:babajob"
                    href="http://www.babajob.com"
                  >
                    BabaJob
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — India
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:bdjobs"
                    href="http://www.bdjobs.com"
                  >
                    Bdjobs
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — Bangladesh
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:brighter+monday"
                    href="https://www.brightermonday.com"
                  >
                    Brighter Monday
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — E Africa
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:catho"
                    href="http://www.catho.com.br"
                  >
                    Catho
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — Brazil
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:jobberman"
                    href="http://www.jobberman.com"
                  >
                    Jobberman
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — W Africa
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:jobsdb"
                    href="http://www.jobsdb.com"
                  >
                    jobsDB
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — SE Asia
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:jobstreet"
                    href="http://www.jobstreet.com"
                  >
                    JobStreet
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — SE Asia
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:jora+int"
                    href="https://www.jora.com"
                  >
                    Jora
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — Worldwide
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:manager"
                    href="http://www.manager.com.br"
                  >
                    Manager
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — Brazil
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:occ+mundial"
                    href="https://www.occ.com.mx"
                  >
                    OCC Mundial
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — Mexico
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:workana"
                    href="https://www.workana.com"
                  >
                    Workana
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — Latin America
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:zhaopin"
                    href="http://www.zhaopin.com"
                  >
                    Zhaopin
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — China
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:jora+au"
                    href="https://au.jora.com"
                  >
                    Jora
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — Australia
                  </span>
                </li>
              </ul>
            </li>
            <li>
              <input
                className="toggle"
                data-automation="partner-services-toggle"
                id="PartnerServicesToggle"
                type="checkbox"
              />
              <label
                className="toggleLink toggleLabel"
                htmlFor="PartnerServicesToggle"
              >
                Partner services
                <span
                  className="root root down chevron"
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "mock svg",
                    }
                  }
                />
              </label>
              <ul
                className="toggleContainer"
              >
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:jobadder"
                    href="https://jobadder.com/au"
                  >
                    JobAdder
                  </a>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:jora+local"
                    data-automation="jora-local-link"
                    href="http://www.joralocal.com.au"
                  >
                    Jora Local
                  </a>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:realestate+au"
                    href="https://www.realestate.com.au/buy"
                  >
                    realestate.com.au
                  </a>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:sidekicker"
                    href="https://www.sidekicker.com.au/"
                  >
                    Sidekicker
                  </a>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:ximble"
                    href="https://www.ximble.com"
                  >
                    Ximble
                  </a>
                </li>
              </ul>
            </li>
          </ul>
        </nav>
        <nav
          aria-label="Connect"
          className="category"
        >
          <h1
            className="heading"
          >
            Connect
          </h1>
          <ul>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:contact+us"
                href="/contact-us"
              >
                Contact us / FAQs
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:product+blog"
                href="http://productblog.seek.com.au"
              >
                Product & Tech Blog
              </a>
            </li>
            <li>
              <input
                className="toggle"
                data-automation={undefined}
                id="SocialToggle"
                type="checkbox"
              />
              <label
                className="toggleLink toggleLabel"
                htmlFor="SocialToggle"
              >
                Social
                <span
                  className="root root down chevron"
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "mock svg",
                    }
                  }
                />
              </label>
              <ul
                className="toggleContainer"
              >
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:facebook"
                    href="https://www.facebook.com/SEEK/"
                  >
                    Facebook
                  </a>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:instagram"
                    href="https://www.instagram.com/seekau/"
                  >
                    Instagram
                  </a>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:twitter"
                    href="https://twitter.com/seekjobs"
                  >
                    Twitter
                  </a>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:youtube"
                    href="https://www.youtube.com/user/SEEKJobs"
                  >
                    YouTube
                  </a>
                </li>
              </ul>
            </li>
          </ul>
        </nav>
        <nav
          aria-label="Employers"
          className="category"
        >
          <h1
            className="heading"
          >
            Employers
          </h1>
          <ul>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:register+for+free"
                href="https://talent.seek.com.au/"
              >
                Register for free
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:post+a+job+ad"
                href="https://talent.seek.com.au/"
              >
                Post a job ad
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:products+prices"
                href="https://talent.seek.com.au/Support/OurProducts"
              >
                Products & prices
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:customer+service"
                href="https://talent.seek.com.au/ContactUs"
              >
                Customer service
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:insight+blog"
                href="http://insightsresources.seek.com.au"
              >
                Insights & Resources
              </a>
            </li>
          </ul>
        </nav>
        <nav
          className="copyright"
        >
          <span
            className="root"
          >
            <h1>
              Additional Links
            </h1>
          </span>
          <a
            className="copyrightLink"
            data-analytics="toolbar:privacy"
            href="/privacy"
          >
            Privacy
          </a>
          <a
            className="copyrightLink"
            data-analytics="toolbar:terms+conditions"
            href="/terms"
          >
            Terms & Conditions
          </a>
          <a
            className="copyrightLink secondaryLink"
            data-analytics="toolbar:seek+safely"
            href="/safe-job-searching"
          >
            SEEK safely
          </a>
          <a
            className="copyrightLink secondaryLink"
            data-analytics="toolbar:site+map"
            href="/sitemap"
          >
            Site Map
          </a>
          <p
            className="copyrightMessage"
          >
            © SEEK. All rights reserved.
          </p>
        </nav>
      </div>
    </section>
  </span>
</footer>
`;

exports[`Footer: should render with locale of NZ 1`] = `
<footer
  aria-labelledby="FooterHeading"
  className="root"
  role="contentinfo"
>
  <span
    className="print"
  >
    <section>
      <span
        className="root"
      >
        <h1
          id="FooterHeading"
        >
          Footer
        </h1>
      </span>
      <div
        className="content"
      >
        <nav
          aria-label="Tools"
          className="category"
        >
          <h1
            className="heading"
          >
            Tools
          </h1>
          <ul>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:profile"
                href="/profile/"
              >
                Profile
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:my+activity"
                href="/myactivity#favourite"
              >
                Saved searches
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:favourite"
                href="/my-activity/saved-jobs"
              >
                Saved jobs
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:applied"
                href="/my-activity/applied-jobs"
              >
                Applied jobs
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:advice+tips"
                href="/career-advice/"
              >
                Advice & tips
              </a>
            </li>
          </ul>
        </nav>
        <nav
          aria-label="Company"
          className="category"
        >
          <h1
            className="heading"
          >
            Company
          </h1>
          <ul>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:about+seek"
                href="/investor/about-us"
              >
                About SEEK
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:media"
                href="/investor/news-media"
              >
                Media
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:work+for+seek"
                href="/work-for-seek"
              >
                Work for SEEK
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:investor+centre"
                href="https://ir.seek.com.au"
              >
                Investor centre
              </a>
            </li>
            <li>
              <input
                className="toggle"
                data-automation={undefined}
                id="InternationalPartnersToggle"
                type="checkbox"
              />
              <label
                className="toggleLink toggleLabel"
                htmlFor="InternationalPartnersToggle"
              >
                International partners
                <span
                  className="root root down chevron"
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "mock svg",
                    }
                  }
                />
              </label>
              <ul
                className="toggleContainer"
              >
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:babajob"
                    href="http://www.babajob.com"
                  >
                    BabaJob
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — India
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:bdjobs"
                    href="http://www.bdjobs.com"
                  >
                    Bdjobs
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — Bangladesh
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:brighter+monday"
                    href="https://www.brightermonday.com"
                  >
                    Brighter Monday
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — E Africa
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:catho"
                    href="http://www.catho.com.br"
                  >
                    Catho
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — Brazil
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:jobberman"
                    href="http://www.jobberman.com"
                  >
                    Jobberman
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — W Africa
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:jobsdb"
                    href="http://www.jobsdb.com"
                  >
                    jobsDB
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — SE Asia
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:jobstreet"
                    href="http://www.jobstreet.com"
                  >
                    JobStreet
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — SE Asia
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:jora+int"
                    href="https://www.jora.com"
                  >
                    Jora
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — Worldwide
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:manager"
                    href="http://www.manager.com.br"
                  >
                    Manager
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — Brazil
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:occ+mundial"
                    href="https://www.occ.com.mx"
                  >
                    OCC Mundial
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — Mexico
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:workana"
                    href="https://www.workana.com"
                  >
                    Workana
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — Latin America
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:zhaopin"
                    href="http://www.zhaopin.com"
                  >
                    Zhaopin
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — China
                  </span>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:jora+au"
                    href="https://au.jora.com"
                  >
                    Jora
                  </a>
                  <span
                    className="partnerCountry"
                  >
                     — Australia
                  </span>
                </li>
              </ul>
            </li>
          </ul>
        </nav>
        <nav
          aria-label="Connect"
          className="category"
        >
          <h1
            className="heading"
          >
            Connect
          </h1>
          <ul>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:contact+us"
                href="/contact-us"
              >
                Contact us / FAQs
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:product+blog"
                href="http://productblog.seek.com.au"
              >
                Product & Tech Blog
              </a>
            </li>
            <li>
              <input
                className="toggle"
                data-automation={undefined}
                id="SocialToggle"
                type="checkbox"
              />
              <label
                className="toggleLink toggleLabel"
                htmlFor="SocialToggle"
              >
                Social
                <span
                  className="root root down chevron"
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "mock svg",
                    }
                  }
                />
              </label>
              <ul
                className="toggleContainer"
              >
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:facebook"
                    href="https://www.facebook.com/seeknz/"
                  >
                    Facebook
                  </a>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:instagram"
                    href="https://www.instagram.com/seeknz/"
                  >
                    Instagram
                  </a>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:twitter"
                    href="https://twitter.com/seekjobsnz"
                  >
                    Twitter
                  </a>
                </li>
                <li
                  className={undefined}
                >
                  <a
                    className="link"
                    data-analytics="toolbar:youtube"
                    href="https://www.youtube.com/user/SEEKJobs"
                  >
                    YouTube
                  </a>
                </li>
              </ul>
            </li>
          </ul>
        </nav>
        <nav
          aria-label="Employers"
          className="category"
        >
          <h1
            className="heading"
          >
            Employers
          </h1>
          <ul>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:register+for+free"
                href="https://talent.seek.co.nz/"
              >
                Register for free
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:post+a+job+ad"
                href="https://talent.seek.co.nz/"
              >
                Post a job ad
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:products+prices"
                href="https://talent.seek.co.nz/Support/OurProducts"
              >
                Products & prices
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:customer+service"
                href="https://talent.seek.co.nz/ContactUs"
              >
                Customer service
              </a>
            </li>
            <li
              className={undefined}
            >
              <a
                className="link"
                data-analytics="toolbar:insight+blog"
                href="http://insightsresources.seek.co.nz"
              >
                Insights & Resources
              </a>
            </li>
          </ul>
        </nav>
        <nav
          className="copyright"
        >
          <span
            className="root"
          >
            <h1>
              Additional Links
            </h1>
          </span>
          <a
            className="copyrightLink"
            data-analytics="toolbar:privacy"
            href="/privacy"
          >
            Privacy
          </a>
          <a
            className="copyrightLink"
            data-analytics="toolbar:terms+conditions"
            href="/terms"
          >
            Terms & Conditions
          </a>
          <a
            className="copyrightLink secondaryLink"
            data-analytics="toolbar:seek+safely"
            href="/safe-job-searching"
          >
            SEEK safely
          </a>
          <a
            className="copyrightLink secondaryLink"
            data-analytics="toolbar:site+map"
            href="/sitemap"
          >
            Site Map
          </a>
          <p
            className="copyrightMessage"
          >
            © SEEK. All rights reserved.
          </p>
        </nav>
      </div>
    </section>
  </span>
</footer>
`;
